README.md

```diff
@@ -23,14 +23,35 @@ Role Variables
 * `control`: whether to enable control host
 * `batch`: whether to enable compute nodes 
 * `runtime`: whether to enable OpenHPC runtime
+* `drain`: whether to drain compute nodes
+* `resume`: whether to resume compute nodes
 
-Example Playbook
+Example Inventory
+-----------------
+
+And an Ansible inventory as this:
+
+    [openhpc_login]
+    openhpc-login-0 ansible_host=10.60.253.40 ansible_user=centos
+
+    [openhpc_compute]
+    openhpc-compute-0 ansible_host=10.60.253.31 ansible_user=centos
+    openhpc-compute-1 ansible_host=10.60.253.32 ansible_user=centos
+
+    [cluster_control:children]
```

Comment:
Do we need to create the `cluster_login` group in the inventory too?

Reply:
That may be good, for consistency.